README.md

```diff
@@ -87,23 +87,23 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
 [rl-arm64]: https://github.com/PowerShell/PowerShell/releases/download/v7.0.0/powershell-7.0.0-linux-arm64.tar.gz
```

Comment:
@TravisEz13, your last commit had 2 failures in `PowerShell-CI-macos`
Requires tests.Version checks.Check for version 7.1
```powershell
Expected an exception, with type [System.Management.Automation.ScriptRequiresException] to be thrown, but no exception was thrown.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Language/Scripting/Requires.Tests.ps1: line 90
90:             { . $File } | Should -Throw -ExceptionType ([System.Management.Automation.ScriptRequiresException])
```
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '127.0.0.1', but got 10.79.1.239.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 257
257:             $result[0].TargetAddress | Should -BeExactly $realAddress
```